python/mlc_chat/compiler/compile.py

```diff
@@ -130,6 +130,14 @@ def _emit_metadata(metadata):
 def _compile(args: CompileArgs, model_config: ConfigBase):
     logger.info("Creating model from: %s", args.config)
     args.overrides.apply(model_config)
+    with args.target:
+        model, _ = args.model.quantize[args.quantization.kind](model_config, args.quantization)
+        logger.info("Exporting the model to TVM Unity compiler")
+        mod, named_params = model.export_tvm(
+            spec=model.get_default_spec(),  # type: ignore
+        )
+    logger.info("Running optimizations using TVM Unity")
+    _attach_variable_bounds(mod, model_config)
```

Comment:
merge this `with` block into the next one, also remove this duplicated `_attach_variable_bounds`